src/enclave/client_endpoint.h

```diff
@@ -17,12 +17,12 @@ namespace enclave
 
     HandleDataCallback handle_data_cb;
 
-    size_t session_id;
+    int64_t session_id;
```

Comment:
Our `session_id`s were a mix of `size_t` and `int64_t` - mostly signed on the host-side and unsigned in the enclave. This was an unnecessary complication. In fact we have 2 classes of sessions - originating inside and outside - and these were already distinguished by the high-bit. But in-enclave we treated these as "very large unsigneds", when they're more simply "negative numbers".